contrib/debian/bitcoin-qt.desktop

```diff
@@ -10,5 +10,4 @@ Terminal=false
 Type=Application
 Icon=bitcoin128
 MimeType=x-scheme-handler/bitcoin;
-Categories=Office;Finance;P2P;Network;Qt;
-StartupWMClass=Bitcoin-qt
+Categories=Office;Finance;
```

Comment:
Note that the additional categories were just added in #12854

Reply:
@fanquake thanks. I'll get that sorted out here...